src/box/lua/net_box.c

```diff
@@ -820,6 +830,26 @@ netbox_encode_select(lua_State *L, int idx, struct mpstream *stream,
 	mpstream_encode_uint(stream, IPROTO_LIMIT);
 	mpstream_encode_uint(stream, limit);
 
+	/* encode after */
+	if (lua_isstring(L, idx + 6)) {
+		mpstream_encode_uint(stream, IPROTO_AFTER_POSITION);
+		size_t size;
+		const char *pos = lua_tolstring(L, idx + 6, &size);
+		char *buf = mpstream_reserve(stream, size);
+		memcpy(buf, pos, size);
+		mpstream_advance(stream, size);
+	} else if (luaT_istuple(L, idx + 6) != NULL ||
+		   lua_istable(L, idx + 6)) {
+		mpstream_encode_uint(stream, IPROTO_AFTER_TUPLE);
+		luamp_encode_tuple(L, cfg, stream, idx + 6);
+	}
+
+	/* encode fetch_pos */
+	if (fetch_pos) {
+		mpstream_encode_uint(stream, IPROTO_FETCH_POSITION);
+		mpstream_encode_bool(stream, fetch_pos);
```

Comment:
What happens if the other end doesn't support pagination? I guess the new arguments are going to be silently ignored, resulting in an invalid select result from the client's pov. I think we should add a new [IPROTO feature](https://github.com/tarantool/tarantool/blob/master/src/box/iproto_features.h) to ensure this doesn't happen.

Reply:
Done